### PR TITLE
feat(deploy.sh): push deisci/workflow:v2-alpha images from master

### PR DESCRIPTION
**Pull request by mboersma**
This is the minimal amount of automation necessary to get a Deis v2 component up for testing.

### CODE REVIEW

File: manifests/deis-workflow-rc.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ spec:
     spec:
       containers:
         - name: deis-workflow
-          image: deis/workflow:latest
+          image: quay.io/deis/workflow:v2-alpha
```
Comment (arschles) [inline manifests/deis-workflow-rc.yml]: @mboersma is quay.io the preferred registry for storing our images?

Comment (mboersma) [inline manifests/deis-workflow-rc.yml]: I think for now we should default to quay.io but store the same images at Docker Hub as well.

Comment (arschles) [inline manifests/deis-workflow-rc.yml]: sounds good to me

File: Makefile [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
-IMAGE_PREFIX ?= $(DEV_REGISTRY)/deis/
-IMAGE=$(IMAGE_PREFIX)$(COMPONENT):$(BUILD_TAG)
-SHELL_SCRIPTS=$(wildcard rootfs/bin/*) $(shell find "rootfs" -name '*.sh')
+
+REGISTRY ?= $(DEV_REGISTRY)
+IMAGE_PREFIX ?= deis
+COMPONENT ?= workflow
+BUILD_TAG ?= git-$(shell git rev-parse --short HEAD)
+IMAGE = $(REGISTRY)/$(IMAGE_PREFIX)/$(COMPONENT):$(BUILD_TAG)
+SHELL_SCRIPTS = $(wildcard rootfs/bin/*) $(shell find "rootfs" -name '*.sh') $(shell find "_scripts/ci" -name '*.sh')
```
Comment (sgoings) [inline Makefile]: What's this bad boy for?

Comment (bacongobbler) [inline Makefile]: [shellcheck](https://github.com/deis/workflow/blob/master/Makefile#L73)

Comment (sgoings) [inline Makefile]: :+1: Thanks for feeding my laziness.

**Pull request merged**